app/controllers/courses/accredited_body_controller.rb

```diff
@@ -112,7 +117,7 @@ def build_course_params
     def errors_for_search_query(code, query)
       errors = {}
 
-      if code == "other" && query.length < 3
+      if code == "other" && query.length < 2
```

Comment:
As discussed elsewhere, can we use `other_selected_with_no_autocompleted_code` here?

Reply:
Done!